### ISSUE TITLE
[ BUG ] Receive-FalconScheduledReport produces no files

### ISSUE THREAD

**Issue by bk-cs** [CrowdStrike/psfalcon#140]
**Describe the bug**
`Receive-FalconScheduledReport` asks for a 32-alphanumeric character identifier, but the `result_id` listed under a report execution is a GUID and sha256 hash combination.

**To Reproduce**
```
$ExecutionId = Get-FalconScheduledReport -Execution -Limit 1
$Execution = Get-FalconScheduledReport -Execution -Ids $ExecutionId
Receive-FalconScheduledReport -Id $Execution.result_metadata.result_id
```
**Expected behavior**
The report should be downloaded.

**Environment (please complete the following information):**
 - OS: Windows 10
 - PowerShell: 5.1, 7.1.4
 - PSFalcon: 2.1.5

**Comment by bk-cs** [comment]
At first I assumed this was a problem related to the pattern being used for the identifier, but now I think the `GET /reports/entities/report-executions-download/v1` API returns a `gzip` file instead of `csv` or `json` (which is how it's documented).

**Comment by bk-cs** [comment]
I confirmed that, llthough the documentation for `GET /reports/entities/report-executions-download/v1` lists CSV or Json as the output, the API is designed to send a `gzip`. I'll need to update the command to handle `gzip` instead.

**Comment by bk-cs** [comment]
Resolved with v2.1.6 release.
